llvm/test/Assembler/lround_vec_intrinsic.ll

```diff
@@ -0,0 +1,29 @@
+; Validate that vector types are accepted for llvm.lround/llvm.llround intrinsic
+; RUN: llvm-as < %s -disable-output 2>&1| FileCheck -allow-empty %s
+
+; CHECK-NOT:assembly parsed, but does not verify as correct
+; CHECK-NOT:Intrinsic does not support vectors
```

Comment:
Check the actual output. not checks are fragile, especially of error output

Reply:
Sure Matt.